logstash-core/lib/logstash/output_delegator.rb

```diff
@@ -153,10 +168,14 @@ def busy_workers
     if @threadsafe
       0
     else
-      @workers.size - @worker_queue.size
+      @workers.size - (@worker_queue ? @worker_queue.size : 0)
```

Comment:
I actually don't remember why I made this change. `@worker_queue` should always be defined so I'm wondering if there's a race here? At any rate I'll investigate this.

You're right, this should use `worker_count` as well.